syntax = "proto3";

option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "OrderBookResponseProto";

import "hq/dto/Security.proto";
import "hq/dto/OrderBook.proto";

message OrderBookResponse {
  Security security = 1; //股票定义
  repeated OrderBook orderBookAskList = 2; //卖盘
  repeated OrderBook orderBookBidList = 3; //买盘
  double spreadLevel = 4; // 最小价格单位 沙股有
}